ui.tests/test-module/libs/commons/guideSelectors.js

```diff
@@ -170,7 +170,10 @@ var selectors = {
             editRule : "#EditableToolbar [data-action='editexpression']",
             createRuleButton : "#create-rule-button",
             saveRule : ".exp-Save-Button",
-            closeRuleEditor : ".exp-Close-Button"
+            closeRuleEditor : ".exp-Close-Button",
+            sideToggleButton : "#exp-toggle-sidepanel-button",
+            formObjectsTab : ".exp-sidepanel coral-tabview coral-tab coral-tab-label:first",
```

Comment:
Can we use specific CSS selectors and not rely on coral, this might break on 650